src/main/jhipster/src/main/webapp/app/germplasm-manager/inventory/lot-creation-dialog.component.ts

```diff
@@ -100,7 +109,8 @@ export class LotCreationDialogComponent implements OnInit {
             const lotDepositRequest = {
                 selectedLots: <SearchComposite>({ searchRequest: null, itemIds: lotUUIDs }),
                 notes: this.deposit.notes,
-                depositsPerUnit: {}
+                depositsPerUnit: {},
+                sourceStudy: this.studyId
```

Comment:
Once we rename the `LotDepositRequestDto`.`sourceStudy` to `sourceStudyId`, we should adjust here

Reply:
Done